base/sort.jl

```diff
@@ -933,73 +932,172 @@ end
     Av
 end
 
-
 """
-    sortrows(A; alg::Algorithm=DEFAULT_UNSTABLE, lt=isless, by=identity, rev::Bool=false, order::Ordering=Forward)
+    sortslices(A; dims, alg::Algorithm=DEFAULT_UNSTABLE, lt=isless, by=identity, rev::Bool=false, order::Ordering=Forward)
 
-Sort the rows of matrix `A` lexicographically.
-See [`sort!`](@ref) for a description of possible
-keyword arguments.
+Sort slices of an array `A`. The keyword argument `dims` must be either
+an integer or a tuple of integers specifies along which dimension to take the
+slices.
+
+If `A` is a matrix, `dims=1` will sort rows, `dims=2` will sort columns.
+Note that the default comparison function on one dimensional slices sorts
+lexicographically.
+
+For convenience, `dims=:rows` and `dims=:cols` are supported as alternative
```

Comment:
We don't do this for anything else, why is this function special in that regard?

Reply:
Fair enough, it was requested for convenience, but I think the behavior is consistent enough that we can do without.